contrib/automation_tests/orbit_symbol_locations.py

```diff
@@ -10,7 +10,7 @@
 
 from core.orbit_e2e import E2ETestSuite
 from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
-from test_cases.symbols_tab import ClearSymbolCache, LoadSymbols
+from test_cases.symbols_tab import ClearSymbolCache, WaitForLoadingSymbolsAndCheckModule
```

Comment:
That seem to be unused. Can you just removed the import?

Reply:
Actually it's because I shouldn't have changed it in this test. Thanks for catching!